integration-tests/js-compute/fixtures/app/src/runtime.js

```diff
@@ -0,0 +1,23 @@
+import {
+  pass,
+  ok,
+  strictEqual
+} from "./assertions-throwing.js";
+import { routes } from "./routes.js";
+import { vCpuTime } from "fastly:runtime";
+
+routes.set("/runtime/get-vcpu-ms", () => {
+  const cpuTime = vCpuTime();
+  strictEqual(typeof cpuTime, 'number');
+  ok(cpuTime > 0);
+  ok(cpuTime < 1000);
+  const arr = [];
+  for (let i = 0; i < 1000; i++) {
+    arr.push(i);
+  }
+  const cpuTime2 = vCpuTime();
+  ok(cpuTime2 > cpuTime);
+  ok(cpuTime2 - cpuTime > 1);
+  ok(cpuTime2 - cpuTime < 1000);
```

Comment:
The two comparisons around the upper bound of `1000` might end up being flaky tests. Does it make sense to bump it up a bit more?

Reply:
It's denoted in milliseconds if that helps to clarify?

Reply:
Yep, I'm just worried that the test will drift in execution time, and cause failures if somehow it goes beyond one second. If you're confident that we won't see that, then we can go ahead and merge.

Reply:
I've bumped it to 3000.